> Error: D:\a\abp\abp\modules\identity\src\Volo.Abp.Identity.Blazor\Pages\Identity\RoleManagement.razor.cs(69,95): error CS1002: ; expected [D:\a\abp\abp\modules\identity\src\Volo.Abp.Identity.Blazor\Volo.Abp.Identity.Blazor.csproj]
    0 Warning(s)
    1 Error(s)

--- a/modules/identity/src/Volo.Abp.Identity.Blazor/Pages/Identity/RoleManagement.razor.cs
+++ b/modules/identity/src/Volo.Abp.Identity.Blazor/Pages/Identity/RoleManagement.razor.cs
@@ -64,10 +64,7 @@ public partial class RoleManagement
                     new EntityAction
                     {
                         Text = L["Delete"],
-                        Visible = (data) =>
-                        {
-                            return HasDeletePermission && !data.As<IdentityRoleDto>().IsStatic
-                        },
+                        Visible = (data) => HasDeletePermission && !data.As<IdentityRoleDto>().IsStatic,
                         Clicked = async (data) => await DeleteEntityAsync(data.As<IdentityRoleDto>()),
                         ConfirmationMessage = (data) => GetDeleteConfirmationMessage(data.As<IdentityRoleDto>())
                     }

--- a/modules/identity/src/Volo.Abp.Identity.Blazor/Pages/Identity/UserManagement.razor.cs
+++ b/modules/identity/src/Volo.Abp.Identity.Blazor/Pages/Identity/UserManagement.razor.cs
@@ -162,10 +162,7 @@ public partial class UserManagement
                     new EntityAction
                     {
                         Text = L["Delete"],
-                        Visible = (data) =>
-                        {
-                            return HasDeletePermission && CurrentUser.GetId() != data.As<IdentityUserDto>().Id;
-                        },
+                        Visible = (data) => HasDeletePermission && CurrentUser.GetId() != data.As<IdentityUserDto>().Id,
                         Clicked = async (data) => await DeleteEntityAsync(data.As<IdentityUserDto>()),
                         ConfirmationMessage = (data) => GetDeleteConfirmationMessage(data.As<IdentityUserDto>())
                     }

